sdk/src/pubkey.rs

```diff
@@ -51,6 +51,14 @@ impl Pubkey {
     pub fn new_rand() -> Self {
         Self::new(&rand::random::<[u8; 32]>())
     }
+
+    #[cfg(feature = "program")]
+    pub fn print(&self) {
```

Comment:
just a nit: I think we call this `log()` instead?  We do `sol_log_64()` it instead of `println!` it